### PR TITLE
Complete deployment on master on rename

### PR DESCRIPTION
**Pull request by vicsn**

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -8,6 +8,16 @@
   src: local('Suisse Intl Book'), url(./fonts/SuisseIntl-Book.woff) format('woff');
 }
 
+@font-face {
+  font-family: 'Messer Condensed';
```
Comment (captnseagraves) [inline frontend/src/App.scss]: These are old versions of the files. The current state of master should have the proper references.

File: frontend/src/App.scss [hunk 2]
```diff
@@ -8,6 +8,16 @@
...
+  src: local('Messer Condensed'), url(./fonts/MesserV2.1TRIAL-Condensed.woff) format('woff');
+}
+
+@font-face {
+  font-family: 'Suisse Intl Book';
+  src: local('Suisse Intl Book'), url(./fonts/SuisseIntl-Book.ttf) format('ttf');
```
Comment (captnseagraves) [inline frontend/src/App.scss]: The files that are referenced here are also added in this PR. They should removed from the PR before merging.

File: frontend/src/App.scss (hunk 1 above)
Comment (vicsn) [reply frontend/src/App.scss]: Fixing in a new commit in this PR.

File: frontend/src/App.scss (hunk 2 above)
Comment (vicsn) [reply frontend/src/App.scss]: Fixing in a new commit in this PR.

**Pull request merged**